test/integration/cleanliness_test.rb

```diff
@@ -5,4 +5,10 @@
   it "does not have boolean limit 1 in schema since this breaks mysql" do
     File.read("db/schema.rb").wont_match /\st\.boolean.*limit: 1/
   end
+
+  it "does not include rails-assets-bootstrap" do
+    # make sure rails-assets-bootstrap did not get included by accident (dependency of some other bootstrap thing)
+    # if it is not avoidable see http://stackoverflow.com/questions/7163264
+    File.read('Gemfile.lock').wont_include 'rails-assets-bootstrap '
```

Comment:
mind making this a regex like `rails-assets-bootstrap\w+(` ? It makes the intention a bit more clear imo. Otherwise people (actually me) will be tempted to remove the whitespace when they see it